fl4health/client_managers/fixed_sampling_client_manager.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Optional
+
+from flwr.server.client_manager import SimpleClientManager
+from flwr.server.client_proxy import ClientProxy
+from flwr.server.criterion import Criterion
+
+
+class FixedSamplingClientManager(SimpleClientManager):
+    """Keeps sampling fixed until it's reset"""
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.current_sample: Optional[List[ClientProxy]] = None
+
+    def reset_sample(self) -> None:
+        """Resets the saved sample so self.sample produces a new sample again."""
+        self.current_sample = None
+
+    def sample(
+        self,
+        num_clients: int,
+        min_num_clients: Optional[int] = None,
+        criterion: Optional[Criterion] = None,
+    ) -> List[ClientProxy]:
+        """
+        Return a new client sample for the first time it runs. For subsequent runs,
+        it will return the sample sampling until self.reset_sampling() is called.
```

Comment:
I think maybe you mean to write "...it will return the **same** sampling until..." here, but I'm not certain 😂